### ISSUE TITLE
[FEATURE] Swaping New and Old CS UI

### ISSUE THREAD

**Issue by Siryphas** [witchertrpg-foundryvtt/TheWitcherTRPG#147]
**Is your feature request related to a problem? Please describe.**
My players and I prefer the visuals of the older character sheet and would like to see if there could be a feature to switch back to the older visuals?

**Comment by Stexinator** [comment]
@moliloo

**Comment by moliloo** [comment]
Maybe its possible, but i need to do a research for how to implement it to confirm for you.

But @Siryphas i want a feedback from you, its just the visual you and your players dont like or even some other features i do?
For Example: Life Bar, Stamina and Toxicity; Death Penality buttons; New collum to show Bonus Modifiers from Global Effects or Active effects in Stats

I just want to know because i want improve the sheet and dont make features to make it worse to use it

**Comment by Siryphas** [comment]
It's mostly the visuals. It looks much more simple than it did before. They're happy with some of the new features like the LUCK points having a refresh button and the automation features. But they feel like the visual design is too modern looking, especially compared to the way it looked before.
I understand this is mostly cosmetic, and thus not likely to be a priority, but we did prefer the look of the last iteration of the character sheet more than the current.

**Comment by Stexinator** [comment]
So maybe separate themes make sense. Something like a more medvieval theme and a clean theme

**Comment by Siryphas** [comment]
Yes, if we could somehow get the new features in a style that looks like the older theme, that would be ideal.
